c_glib/arrow-glib/decimal.cpp

```diff
@@ -0,0 +1,185 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include <config.h>
+#endif
+
+#include <arrow-glib/decimal.hpp>
+
+G_BEGIN_DECLS
+
+/**
+ * SECTION: decimal128
+ * @title: Decimal128 class
+ * @include: arrow-glib/arrow-glib.h
+ *
+ * #GArrowDecimal128 is a decimal128 class.
+ *
+ * Since: 0.9.0
```

Comment:
Can you use `0.10.0`?
Because we will ship this feature in Apache Arrow GLib 0.10.0.